account_general_ledger_report/wizard/general_ledger_wizard.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="account_line_tree_view" model="ir.ui.view">
+            <field name="name">account.line.tree.view</field>
+            <field name="model">account.account</field>
+            <field name="arch" type="xml">
+                <tree string="GL Lines" editable="bottom">
+                    <field name="code"/>
+                    <field name="name"/>
+                    <field name="user_type"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_account_general_ledger_wizard" model="ir.ui.view">
+            <field name="name">view.account.general.ledger.wizard</field>
+            <field name="model">account.general.ledger.wizard</field>
+            <field name="arch" type="xml">
+                <form string="Report Options">
+                    <group string="General Ledger (web)">
+                        <group>
+                            <field name="fiscalyear_id"/>
+                            <field name="account_ids" widget="many2many_tags"
+                              context="{'tree_view_ref': 'account_general_ledger_report.account_line_tree_view'}"/>
+                            <!-- create space -->
+                            <field name="field_free" readonly="True" string=" "/>
```

Comment:
You can use html, i.e., <br/><br/>
No need to create unnecessary field (it is not beautiful code)